compiler/passes/normalize.cpp

```diff
@@ -1744,6 +1764,14 @@ static void change_method_into_constructor(FnSymbol* fn) {
     }
   }
 
+  if (ct->instantiationStyle == DEFINES_INITIALIZER) {
+    ArgSymbol* meme = new ArgSymbol(INTENT_BLANK, "meme", ct, NULL,
```

Comment:
Ehhhhhh, that seems too special cases for my taste, but others on the team might feel differently.